test/Dialect/FIRRTL/annotations.fir

```diff
@@ -380,57 +380,6 @@ circuit Foo: %[[{"target": "~Foo|Foo>_T_0", "a": "a"},
 
 ; // -----
 
-; DontTouch annotation preserves temporary names
-circuit Foo: %[[{"target": "~Foo|Foo>_T_0", "class": "firrtl.transforms.DontTouchAnnotation"},
-                {"target": "~Foo|Foo>_T_1", "class": "firrtl.transforms.DontTouchAnnotation"},
-                {"target": "~Foo|Foo>_T_2", "class": "firrtl.transforms.DontTouchAnnotation"},
-                {"target": "~Foo|Foo>_T_3", "class": "firrtl.transforms.DontTouchAnnotation"},
-                {"target": "~Foo|Foo>_T_4", "class": "firrtl.transforms.DontTouchAnnotation"},
-                {"target": "~Foo|Foo>_T_5", "class": "firrtl.transforms.DontTouchAnnotation"},
-                {"target": "~Foo|Foo>_T_6", "class": "firrtl.transforms.DontTouchAnnotation"},
-                {"target": "~Foo|Foo>_T_7", "class": "firrtl.transforms.DontTouchAnnotation"},
-                {"target": "~Foo|Foo>_T_8", "class": "firrtl.transforms.DontTouchAnnotation"},
-                {"target": "~Foo|Foo>_T_9.a", "class": "firrtl.transforms.DontTouchAnnotation"}]]
-  module Bar:
-    skip
-  module Foo:
-    input reset : UInt<1>
-    input clock : Clock
-
-    ; CHECK: %_T_0 = firrtl.wire sym @_T_0
-    wire _T_0 : UInt<1>
-    ; CHECK: %_T_1 = firrtl.node sym @_T_1 %_T_0
-    node _T_1 = _T_0
-    ; CHECK: %_T_2 = firrtl.reg sym @_T_2 %clock
-    reg _T_2 : UInt<1>, clock
-    ; CHECK: %_T_3 = firrtl.regreset sym @_T_3 %clock, %reset, %c0_ui4
-    reg _T_3 : UInt<4>, clock with :
-      reset => (reset, UInt<4>("h0"))
-    ; CHECK: %_T_4 = chirrtl.seqmem sym @_T_4 Undefined
-    smem _T_4 : UInt<1>[9] [256]
-    ; CHECK: %_T_5 = chirrtl.combmem sym @_T_5
-    cmem _T_5 : UInt<1>[9] [256]
-    ; CHECK: chirrtl.memoryport Infer %_T_5 {annotations =
-    ; CHECK-SAME: {class = "firrtl.transforms.DontTouchAnnotation"}
-    infer mport _T_6 = _T_5[reset], clock
-    ; CHECK: firrtl.instance _T_7 sym @_T_7
-    inst _T_7 of Bar
```

Comment:
I dropped this part of this test as SFC will not let you put a `DontTouchAnnotation` on a module or instance.